### PR TITLE
Fix Merge Conflict for namespace

### PR DESCRIPTION
**Pull request by dungeonmaster18**
**What this PR does / why we need it**:
Fixes Merge Conflict on master branch.

**Which issue this PR fixes** : fixes [openebs/openebs #1066](https://github.com/openebs/openebs/issues/1066)

**Special notes for your reviewer**:
/cc @kmova @AmitKumarDas

### CODE REVIEW

File: orchprovider/k8s/v1/k8s.go [hunk 1]
```diff
@@ -742,7 +838,7 @@ func (k *k8sOrchestrator) createControllerDeployment(volProProfile volProfile.Vo
 	}
 
 	if !imgSupport {
-		return nil, fmt.Errorf("Volume '%s' requires a controller container image", vsm)
+		return nil, fmt.Errorf("VSM '%s' requires a controller container image", vsm)
```
Comment (prateekpandey14) [inline orchprovider/k8s/v1/k8s.go]: same as above

File: orchprovider/k8s/v1/k8s.go [hunk 2]
```diff
@@ -733,7 +829,7 @@ func (k *k8sOrchestrator) createControllerDeployment(volProProfile volProfile.Vo
 	}
 
 	if clusterIP == "" {
-		return nil, fmt.Errorf("Volume cluster IP is required to create controller for volume 'name: %s'", vsm)
+		return nil, fmt.Errorf("VSM cluster IP is required to create controller for vsm 'name: %s'", vsm)
```
Comment (prateekpandey14) [inline orchprovider/k8s/v1/k8s.go]: change to volume

File: orchprovider/k8s/v1/k8s.go [hunk 3]
```diff
@@ -759,7 +855,7 @@ func (k *k8sOrchestrator) createControllerDeployment(volProProfile volProfile.Vo
 		return nil, err
 	}
 
-	glog.Infof("Adding controller for volume 'name: %s'", vsm)
+	glog.Infof("Adding controller for VSM 'name: %s'", vsm)
```
Comment (prateekpandey14) [inline orchprovider/k8s/v1/k8s.go]: change to volume

File: pkg/client/mapiserver/snapshot.go [hunk 4]
```diff
@@ -165,7 +167,7 @@ func ListSnapshot(volName string) error {
 	if code != http.StatusOK {
 		return fmt.Errorf("Server status error: %v", http.StatusText(code))
 	}
-	snapdisk, err := getInfo(body)
+	snapdisk, err := getInfo([]byte(body))
```
Comment (prateekpandey14) [inline pkg/client/mapiserver/snapshot.go]: this is not required. ReadAll method returns []byte which is getting passed to this method.

File: pkg/client/mapiserver/snapshot.go [hunk 5]
```diff
@@ -139,7 +140,8 @@ func ListSnapshot(volName string) error {
 
 	_, err := GetStatus()
 	if err != nil {
-		return fmt.Errorf("Unable to contact maya-apiserver: %s", GetURL())
+		err := errors.New(fmt.Sprintf("Unable to contact maya-apiserver: %s", GetURL()))
```
Comment (prateekpandey14) [inline pkg/client/mapiserver/snapshot.go]: errors.New() does the same as fmt.Errorf() so not required here.

File: pkg/client/mapiserver/snapshot.go [hunk 6]
```diff
@@ -91,7 +92,7 @@ func RevertSnapshot(volName string, snapName string) error {
 
 	_, err := GetStatus()
 	if err != nil {
-		err := fmt.Errorf("Unable to contact maya-apiserver: %s", GetURL())
+		err := errors.New(fmt.Sprintf("Unable to contact maya-apiserver: %s", GetURL()))
```
Comment (prateekpandey14) [inline pkg/client/mapiserver/snapshot.go]: errors.New() does the same as fmt.Errorf() so not required here.

File: pkg/client/mapiserver/snapshot.go [hunk 7]
```diff
@@ -39,7 +40,7 @@ func CreateSnapshot(volName string, snapName string) error {
...
```
Comment (prateekpandey14) [inline pkg/client/mapiserver/snapshot.go]: errors.New() does the same as fmt.Errorf() so not required here.

**Comment by dungeonmaster18** [comment]
Did the changes @prateekpandey14 PTAL.

File: orchprovider/k8s/v1/k8s.go [hunk 8]
```diff
@@ -1388,23 +1484,25 @@ func (k *k8sOrchestrator) getDeploymentList(vsm string, volProProfile volProfile
 	}
 
 	if deployList == nil {
-		return nil, fmt.Errorf("Volume(s) '%s:%s' not found at orchestrator '%s:%s'", ns, vsm, k.Label(), k.Name())
+		return nil, fmt.Errorf("VSM(s) '%s:%s' not found at orchestrator '%s:%s'", ns, vsm, k.Label(), k.Name())
```
Comment (prateekpandey14) [inline orchprovider/k8s/v1/k8s.go]: nit: Change to volume

File: orchprovider/k8s/v1/k8s.go [hunk 9]
```diff
@@ -929,7 +1025,7 @@ func (k *k8sOrchestrator) createReplicaDeployment(volProProfile volProfile.Volum
 	//for rcIndex := 1; rcIndex <= rCount; rcIndex++ {
 	//glog.Infof("Adding replica #%d for VSM '%s'", rcIndex, vsm)
 
-	glog.Infof("Adding replica(s) for Volume '%s'", vsm)
+	glog.Infof("Adding replica(s) for VSM '%s'", vsm)
```
Comment (prateekpandey14) [inline orchprovider/k8s/v1/k8s.go]: nit: Change to volume

File: orchprovider/k8s/v1/k8s.go [hunk 10]
```diff
@@ -885,7 +981,7 @@ func (k *k8sOrchestrator) createReplicaDeployment(volProProfile volProfile.Volum
 	}
 
 	if clusterIP == "" {
-		return nil, fmt.Errorf("Volume cluster IP is required to create replica(s) for volume 'name: %s'", vsm)
+		return nil, fmt.Errorf("VSM cluster IP is required to create replica(s) for vsm 'name: %s'", vsm)
```
Comment (prateekpandey14) [inline orchprovider/k8s/v1/k8s.go]: nit: Change to volume

**Comment by prateekpandey14** [comment]
@dungeonmaster18 some more comments related to naming `vsm --> volume`, please have a look.

**Comment by dungeonmaster18** [comment]
@prateekpandey14 it seems #234 has VSM. @AmitKumarDas  told me to do changes according to that. So I am bit confuse now.

**Comment by dungeonmaster18** [comment]
@prateekpandey14 it seems #234 has VSM. @AmitKumarDas  told me to do changes according to that. So I am bit confuse now.

**Comment by prateekpandey14** [comment]
Thanks @dungeonmaster18 you are right, but master branch contains all the changes as `volume` now
so thought was to sync everything properly, same conventions.

**Comment by dungeonmaster18** [comment]
So I have to change all VSM to volume

**Comment by prateekpandey14** [comment]
Yes , please.

**Comment by dungeonmaster18** [comment]
Cool. I will do that by evening and ping you.

**Comment by dungeonmaster18** [comment]
@prateekpandey14 PTAL

**Pull request merged**